crates/typst-layout/src/math/mod.rs

```diff
@@ -58,12 +57,20 @@ pub fn layout_equation_inline(
 
     let mut locator = locator.split();
     let mut ctx = MathContext::new(engine, &mut locator, styles, region, &font);
+
+    let local = [
+        EquationElem::set_script_scale(ctx.constants.script_percent_scale_down()),
+        EquationElem::set_sscript_scale(ctx.constants.script_script_percent_scale_down()),
```

Comment:
this could even be a single pair entry for a bit less runtime cost

Reply:
That seems reasonable. I will modify it.